earth2mip/lagged_ensembles/score.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import torch
+
+import earth2mip.grid
+from earth2mip.crps import crps_from_empirical_cdf
+
+logger = logging.getLogger(__name__)
+
+
+def weighted_average(x, w, dim):
+    return torch.mean(x * w, dim) / torch.mean(w, dim)
+
+
+def area_average(grid: earth2mip.grid.LatLonGrid, x: torch.Tensor):
+    lat = torch.tensor(grid.lat, device=x.device)[:, None]
+    cos_lat = torch.cos(torch.deg2rad(lat))
+    return weighted_average(x, cos_lat, dim=[-2, -1])
```

Comment:
Why is this here? Modulus has lat and global averaging utilities.

Reply:
See reply here: https://github.com/NVIDIA/earth2mip/pull/154#discussion_r1440820519